### PR TITLE
Update ru.po

### PR DESCRIPTION
**Pull request by Alexmelman88**

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -2151,20 +2152,20 @@ msgstr "Архитектура, которая станет текущей дл
 
 #: app/flatpak-builtins-make-current.c:58
 msgid "APP BRANCH - Make branch of application current"
-msgstr "ПРИЛОЖЕНИЕ ВЕТКА - Сделать ветку текущей для приложения"
+msgstr "ВЕТВЬ ПРИЛОЖЕНИЯ - Сделать ветвь приложения текущей"
```
Comment (gasinvein) [inline po/ru.po:2155]: `APP` and `BRANCH` here are placeholders for cli args, and they must be in this order.

File: po/ru.po [hunk 2]
```diff
@@ -426,7 +426,7 @@ msgstr "Имя архитектуры для экспорта (должна бы
 
 #: app/flatpak-builtins-build-export.c:62
 msgid "Commit runtime (/usr), not /app"
-msgstr "Зафиксировать среду исполнения (/usr) вместо приложения (/app)"
+msgstr "Коммит среды выполнения (/usr), не /app"
```
Comment (gasinvein) [inline po/ru.po:429]: It should be an imperative sentence ("Commit" here is a verb, not noun).

File: po/ru.po [hunk 3]
```diff
@@ -3126,7 +3127,7 @@ msgstr "ПРИЛОЖЕНИЕ [АРГУМЕНТ…] - Запустить прил
 #: app/flatpak-builtins-run.c:264
 #, c-format
 msgid "runtime/%s/%s/%s not installed"
-msgstr "runtime/%s/%s/%s не установлено"
+msgstr "среда выполнения/%s/%s/%s не установлена"
```
Comment (gasinvein) [inline po/ru.po:3130]: `runtime/` is a ref prefix, it's not translatable.

File: po/ru.po [hunk 4]
```diff
@@ -1917,12 +1916,14 @@ msgstr "Не использовать статические дельты"
 
 #: app/flatpak-builtins-install.c:74
 msgid "Additionally install the SDK used to build the given refs"
-msgstr ""
+msgstr "Дополнительно установите SDK, используемый для создания данных ссылок"
```
Comment (gasinvein) [inline po/ru.po:1919]:
```suggestion
msgstr "Дополнительно установить SDK, используемый для создания данных ссылок"
```

File: po/ru.po [hunk 5]
```diff
@@ -1917,12 +1916,14 @@ msgstr "Не использовать статические дельты"
...
 
 #: app/flatpak-builtins-install.c:75
 msgid ""
 "Additionally install the debug info for the given refs and their dependencies"
 msgstr ""
+"Дополнительно установите отладочную информацию для заданных ссылок и их "
```
Comment (gasinvein) [inline po/ru.po:1925]:
```suggestion
"Дополнительно установить отладочную информацию для заданных ссылок и их "
```

File: po/ru.po [hunk 6]
```diff
@@ -355,7 +355,7 @@ msgstr "ЗНАЧ"
 
 #: app/flatpak-builtins-build-commit-from.c:72
 msgid "Override the timestamp of the commit (NOW for current time)"
-msgstr "Переназначить отметку времени коммита (NOW для текущего времени)"
+msgstr "Переназначить отметку времени коммита (СЕЙЧАС для текущего времени)"
```
Comment (gasinvein) [inline po/ru.po:358]: `NOW` here is a special value, not translatable.

**Review by gasinvein — CHANGES_REQUESTED** [review]

**Comment by Alexmelman88** [comment]
fixed

**Review by gasinvein — COMMENTED** [review]
Since you've changed the "commit" and "branch" words translations, please make sure you've changed these words everywhere, for consistency. Other than this, seems fine.

**Comment by Alexmelman88** [comment]
> Since you've changed the "commit" and "branch" words translations, please make sure you've changed these words everywhere, for consistency. Other than this, seems fine.

I have changed them everywhere for consistency :)

**Pull request merged**